Documentation/ceph-backup.md

```diff
@@ -0,0 +1,483 @@
+export AWS_ENDPOINT=localhost:8080
+export BUCKET_NAME=`kubectl get configmap -n rook-ceph ceph-delete-bucket -o 'jsonpath={.data.BUCKET_NAME}';echo`
+export AWS_ACCESS_KEY_ID=`kubectl get secret -n rook-ceph ceph-delete-bucket -o 'jsonpath={.data.AWS_ACCESS_KEY_ID}' | base64 --decode;echo`
+export AWS_SECRET_ACCESS_KEY=`kubectl get secret -n rook-ceph ceph-delete-bucket -o 'jsonpath={.data.AWS_SECRET_ACCESS_KEY}' | base64 --decode;echo`
+
+for i in {1..10};do aws s3 cp /etc/hosts s3://${BUCKET_NAME}/$i --endpoint-url http://${AWS_ENDPOINT};done
+
+# List data
+aws s3 ls s3://${BUCKET_NAME} --endpoint-url http://${AWS_ENDPOINT}
+```
+
+## Install and configure Velero
+
+[Velero](https://velero.io/) is an open source tool to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes.
+
+For backing up and restoring Kubernetes volumes you need to use the [restic plugin](https://velero.io/docs/v1.5/restic/). You need to add `--use-restic` to velero install command.
+
+Velero supports a lot of cloud providers. Take at look in the [supported providers](https://velero.io/docs/v1.5/supported-providers/) section of the doc for more information and examples. 
+
+For example for [GCP](https://github.com/vmware-tanzu/velero-plugin-for-gcp#setup).
+
+```console
+
+# Configure Google Storage bucket and service account.
+BUCKET=<YOUR_BUCKET>
+gsutil mb gs://$BUCKET/
+
+PROJECT_ID=$(gcloud config get-value project)
+
+gcloud iam service-accounts create velero \
+    --display-name "Velero service account"
+
+SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list \
+  --filter="displayName:Velero service account" \
+  --format 'value(email)')
+
+ROLE_PERMISSIONS=(
+    compute.disks.get
+    compute.disks.create
+    compute.disks.createSnapshot
+    compute.snapshots.get
+    compute.snapshots.create
+    compute.snapshots.useReadOnly
+    compute.snapshots.delete
+    compute.zones.get
+)
+
+gcloud iam roles create velero.server \
+    --project $PROJECT_ID \
+    --title "Velero Server" \
+    --permissions "$(IFS=","; echo "${ROLE_PERMISSIONS[*]}")"
+
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member serviceAccount:$SERVICE_ACCOUNT_EMAIL \
+    --role projects/$PROJECT_ID/roles/velero.server
+
+gcloud iam service-accounts keys create credentials-velero \
+    --iam-account $SERVICE_ACCOUNT_EMAIL
+
+# Install velero
+velero install --provider gcp --plugins velero/velero-plugin-for-gcp:v1.1.0 --bucket $BUCKET --secret-file ./credentials-velero --use-restic
+
+```
+
+## Create Back Up
+
+We need to save Volumes, Volumes Claims, ConfigMaps, Secrets and Services.
+
+```console
+velero backup create <BackupName> --include-namespaces rook-ceph --include-resources pv,pvc,services,configmaps,secrets --default-volumes-to-restic
+```
+
+**IMPORTANT:** make sure to back up the bucket's configmaps and secrets that are located in a different namespace.
```

Comment:
Which different namespace? The application namespace?
```suggestion
**IMPORTANT:** In addition to backing up the resources in the `rook-ceph` namespace, make sure to back up the bucket's configmaps and secrets that are located in the application namespace.
```

Reply:
When you execute for example `object-bucket-claim-delete.yaml`. If you set  `namespace`, configmaps and secrets will go to other namespace